release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2550,6 +2602,78 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
 
 // TODO: ^^^ Add BZ when created.
 
+[id="ocp-4-10-ran-ga-known-issues"]
+
+* The Kubelet service monitor scrape interval is currently set to a hard-coded value. This means that there is less available CPU resources for workloads.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2035046[*BZ#2035046*])
+
+* Deploying a single node Openshift cluster for vRAN Distributed Units can take up to 4 hours.
```

Comment:
lgtm

Reply:
SME ack is here https://github.com/openshift/openshift-docs/pull/41683/files#r846179390